doc/test-requirements.txt

```diff
@@ -0,0 +1 @@
+#
```

Comment:
can we drop this file?

Reply:
I guess I used what is the convention, but we can ofc remove it in a later patchset.